extension/content/components/recipes/form/arguments/ShowHeartBeatArguments.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import {
+  HelpBlock,
+  FormGroup,
+  ControlLabel,
+  InputNumber,
+  Grid,
+  Row,
+  Col,
+} from "rsuite";
+
+import {
+  ACTION_UPDATE_DATA,
+  useRecipeDetailsData,
+  useRecipeDetailsDispatch,
+} from "devtools/contexts/recipeDetails";
+
+import InputField from "devtools/components/recipes/form/arguments/fields/InputField";
+import SelectField from "devtools/components/recipes/form/arguments/fields/SelectField";
+import CheckboxField from "devtools/components/recipes/form/arguments/fields/CheckboxField";
+import InputUrlField from "devtools/components/recipes/form/arguments/fields/InputUrlField";
+
+export default function ShowHeartBeatArguments() {
+  return (
+    <Grid fluid>
+      <Row>
+        <Col xs={12}>
+          <InputField label="Survey ID" name="surveyID" />
+          <InputField label="Message" name="message" />
+          <InputField label="Learn More Message" name="learnMoreMessage" />
+          <RepeatOptionsField />
+        </Col>
+        <Col xs={12}>
+          <InputField
+            label="Engagement Button Label"
+            name="engagementButtonLabel"
+          />
+          <InputField label="Thanks Message" name="thanksMessage" />
+          <InputUrlField label="Post-Answer URL" name="postAnswerUrl" />
```

Comment:
I think an `InputField` should be fine for this based on my comments below about not needing the ability to link out from the field.